test-fixtures/urn/v11/input.json

```diff
@@ -0,0 +1,21 @@
+{
+  "leader": "01331cam a22003494i 4500",
+  "fields": [
+    { "tag": "001", "value": "000000001" },
+    { "tag": "020", "ind1": " ", "ind2": " ", "subfields": [ {"code": "a", "value": "978-951-9155-47-0"} ]},
+    { "tag": "337", "ind1": " ", "ind2": " ", "subfields": [ {"code": "b", "value": "c"}, {"code": "2", "value": "rdamedia"} ] },
+    { "tag": "856", "ind1": "4", "ind2": "0", "subfields": [
+      {"code": "u", "value": "http://urn.fi/URN:ISBN:978-951-9155-47-0"},
+      {"code": "5", "value": "FI-Vapaa"}
```

Comment:
Could this subfield be omitted to simulate case where there is one f856 with URN but not $5?

Reply:
Definitely maybe. However, again not my test. Just something it converted from old format to new. Needs to be discussed with whomever made the original test.